public/sw_user_cache.js

```diff
@@ -285,12 +284,6 @@ async function removeUserCache() {
           cache.delete(item);
           userRemoteLinks = userRemoteLinks.filter(link => link !== item.url);
         }
-        // Removes '/profile/firstName.lastName' from the list of user's remote links and cache
-        // since that file is associated with the user.
-        else if (item.url.match(location.origin) && item.url.includes('/profile/')) {
-          cache.delete(item);
-          userRemoteLinks = userRemoteLinks.filter(link => link !== item.url);
-        }
       });
```

Comment:
The code above has been deleted because we no longer cache the user's public profile. Therefore this check is no longer required